src/com/owncloud/android/lib/common/network/WebdavEntry.java

```diff
@@ -210,6 +213,19 @@ public WebdavEntry(MultiStatusResponse ms, String splitElement) {
                 mIsFavorite = false;
             }
 
+            // NC encrypted property <nc:is-encrypted>
+            prop = propSet.get(EXTENDED_PROPERTY_IS_ENCRYPTED,  Namespace.getNamespace(NAMESPACE_NC));
+            if (prop != null) {
+                String encryptedValue = (String) prop.getValue();
+                if ("1".equals(encryptedValue)) {
```

Comment:
Can we make ```"1"``` a constant so this gets the look of a business rule to be read like ```if (IS_ENCRYPTED.equals(encryptedValue))``` ?